Remotion/Core/Reflection/MethodInfoAdapter.cs

```diff
@@ -117,14 +119,15 @@ public ITypeInformation GetOriginalDeclaringType ()
       return AttributeUtility.GetCustomAttributes<T> (_methodInfo, inherited);
     }
 
-    public object Invoke (object instance, object[] parameters)
+    public object Invoke (object? instance, object?[]? parameters)
     {
-      ArgumentUtility.CheckNotNull ("instance", instance);
+      //TODO RM-7432: Implementation requires not null on nullable parameter

```

Comment:
something got lost somewhere in the comments, don't know where, don't know how, don't case.

here and in property info (GetValue, SetValue): the instance can be null, the null-check is wrong. -> otherwise, we don't support statics and that's just wrong.